package/MDAnalysis/lib/distances.py

```diff
@@ -332,6 +341,14 @@ def distance_array(
     distances = _check_result_array(result, (refnum, confnum))
     if len(distances) == 0:
         return distances
+
+    if backend == "distopia":
+        # distopia requires that all the input arrays are the same type,
+        # while MDAnalysis allows for mixed types, this should be changed
+        # pre 0.3.0 release see issue #3707
```

Comment:
Do you mean MDA 3.0.0 release (and not 0.3.0)?

Reply:
Yes fixed